src/components/Search.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+	<div>
+		<h2>Search in list</h2>
+			<div class="form-group">
+			<input class="form-control form-control-lg"
+					type="text"
+					placeholder="type a value..."
+					:value="value"
+					@input="onInput">
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'Search',
+	props: {
+		value: String,
+	},
+	model: {
```

Comment:
Not very convenient way when you work with simple inputs - you need to extract DOM node to emit new value